Index.html

```diff
@@ -214,20 +214,42 @@ <h3>Multi-Post Stories</h3>
                         </ul>
                     
                         <button type="submit">See Project</button>
-                    </div></div>
+                    </div>
+                </div>
             </div>
         </section>
-        <section class="grid-item form">
-            <h4>CONTACT US</h4>
-            <p>If you have an application you are interested in developing, a feature 
-                that you need built or a project that needs coding. I’d love to help with it </p>
-                <form action="" method="post">
-            <input type="text" name="name" placeholder="Text input">
-            <input type="email" name="email" placeholder="Enter your email">
-            <textarea name="comment" id="" cols="30" rows="10" placeholder="Enter your text here"></textarea>
-        </form>
+        <section class="grid-item formContent">
+            <h5>CONTACT US</h5>
+            <p>If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I'd love to help with it </p>
+            <form action="https://formspree.io/f/mrgjqdpe" method="post">
+                    <input class="textInput" type="text" name="name" placeholder="Text input" maxlength="30" required>
+                    <input class="emailInput" type="email" name="email" placeholder="Enter your email" required>
+                    <textarea class="area" name="comment" rows="15" cols="24" maxlength="500" placeholder="Enter your text here" required></textarea>
```

Comment:
- [ ] Good job implementing the form validations 💯. It would be great to add more padding to the input placeholders so that it can more like the design and improve the user interface.
- [ ] Also the height of your `textarea` is far more than that in the Figma template. I think it would be better if you could reduce it a little.
- Your design:
![Annotation 2022-02-23 045405](https://user-images.githubusercontent.com/85113567/155259987-3a5be318-126c-406d-ab5f-66a426a51ff9.jpg)
- Figma design:
![Annotation 2022-02-23 045346](https://user-images.githubusercontent.com/85113567/155260004-232e5bc3-dacd-49df-8070-334f45e7b29c.jpg)